src/components/InstanceFiltersBrowse/InstanceFiltersBrowse.js

```diff
@@ -138,6 +140,32 @@ const InstanceFiltersBrowse = props => {
           {renderSharedFacet(FACETS.SUBJECTS_SHARED)}
           {/* Hide Held by facet for contributors and subjects browse until back-end requirements and implementation are done */}
           {/* {renderHeldByFacet(FACETS.SUBJECTS_HELD_BY)} */}
+          <MultiSelectionFacet
+            id={FACETS.SUBJECT_SOURCE}
+            label={intl.formatMessage({ id: `ui-inventory.${FACETS.SUBJECT_SOURCE}` })}
+            name={FACETS.SUBJECT_SOURCE}
+            closedByDefault
+            separator={checkIfUserInMemberTenant(stripes)}
```

Comment:
Let's show the separator only for ECS for this facet.

Reply:
there is the "Shared" filter above this "Subject source" filter and it is shown only for member tenants, that's why I used `checkIfUserInMemberTenant(stripes)` and not `isConsortiaEnv(stripes)`. So if we show the separator for ECS then for the central tenant it will be redundant